Geometry_Engine/Query/Normal.cs

```diff
@@ -97,17 +107,20 @@ public static List<Vector> Normals(this Mesh3D mesh)
         /**** Public Methods - Curves                   ****/
         /***************************************************/
 
+        [Description("Returns the normal vector to the curve's plane, oriented according to the rigth hand rule. Works only for closed and planar curves.")]
+        [Input("curve", "The Polyline to get the normal to.")]
+        [Input("tolerance", "Distance tolerance used in geometry processing, default set to BH.oM.Geometry.Tolerance.Distance.")]
+        [Output("normal", "The normal vector to the curve's plane.")]
```

Comment:
Minor change, would sound cleared to me (applies to other methods too).
```suggestion
        [Output("normal", "The normal vector to the plane of a curve.")]
```